docs/references/backend/organization/update-organization-metadata.mdx

```diff
@@ -3,9 +3,11 @@ title: updateOrganizationMetadata()
 description: Use Clerk's Backend SDK to update the metadata associated with a given organization.
 ---
 
+{/* clerk/javascript file: https://github.com/clerk/javascript/blob/main/packages/backend/src/api/endpoints/OrganizationApi.ts#L159 */}
+
 # `updateOrganizationMetadata()`
 
-Updates the metadata associated with the specified organization ID.
+Updates the metadata attributes of an [`Organization`](/docs/references/backend/types/backend-organization) by merging existing values with the provided parameters. Metadata values will be updated via a "deep" merge - "deep" meaning that any nested JSON objects will be merged as well. You can remove metadata keys at any level by setting their value to `null`.
```

Comment:
I thought I was rereading the same changes and realized its just the same message in another section 😅 Always good to have the information consistent!